pkg/compiler/codegen.go

```diff
@@ -870,6 +873,10 @@ func (c *codegen) Visit(node ast.Node) ast.Visitor {
 			ast.Walk(c, n.Fun)
 			emit.Opcodes(c.prog.BinWriter, opcode.CALLA)
 		case isSyscall(f):
+			if f.name == "Notify" {
```

Comment:
Check package also?

Reply:
There is a single syscall with that name, but yeah, better check a package also.